type_check_failed,error_msgs
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in check_can_access: No attribute 'organization' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in parse: Name 'values' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in copy: Name 'parser_copy' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in replace_argument: No attribute 'argument_class' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in run_as_module: Name 'code' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in settings: No attribute 'app' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_marshal: Name 'output' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_handle_auth: Name 'api' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in tiff_to_array: Name 'local_raster_path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _iter_conf: Name 'conf' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in score: Name 'v' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _closure_sentinel_attr: Name 'meta' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_descendants: No attribute '_toplevel' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in getGameState: Name 'state' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in main: Name 'th' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in save: Name 'threads' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in save: Name 'user' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in user_data: No attribute 'get_json' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"invalid syntax (<unknown>, line 3)"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in game_over: Name 'obj_loc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in update: No attribute 'body' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in fit: No attribute 'pipeline' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _filereader_mol2: unsupported operand type(s) for item retrieval: 'n: int' and 'slice(None, 1, None): slice' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in residues: unsupported operand type(s) for item retrieval: 'aid: int' and ''path': str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __iter__: No attribute 'Mol' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_soup: Name 'html' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in parse_lxml: Name 'lxml_tree' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in patch: Name 'confvalue' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in run: Name 'location' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_get_version: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in deadline: No attribute '_today' on datetime.timedelta [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in run: No attribute 'function' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in umount: Name 'db_uid' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in drop: Name 'db_uid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_last_sent: Name 'soap_logger' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in format_robot_time: Name 'secs' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Function Logger.addHandler was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in dict_to_options: Name 'v' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _log_function_call: No attribute 'iteritems' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in dry_guard: 'dict' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in _get_transport: 'dict' object is not callable [not-callable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in bench_mget_on_serial_keys: No attribute 'client' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in bench_range: No attribute 'client' on tuple [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 2, in bench_slice: No attribute 'client' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in Slice: No attribute 'CreateSnapshot' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_load: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_drop_existing_db: Name 'store_datas' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_stop_all: No attribute 'assertEqual' on int [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_inactive_worker_last_activity: No attribute 'supervisor' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_inactive_workers_last_activities: unsupported operand type(s) for item retrieval: 'workers_count: int' and '1: int' [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 7, in test_valid_working_command_with_workers: No attribute 'assertGreaterEqual' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'providing_args' on List[nothing] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in get_document: Name 'err' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in get_parser: Name 'module_parser' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in main: Name 'ex' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_save_resource_with_same_name_as_another_resource: Name 'new_resource' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_outgoing_email: Name 'user' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_recent_message_non_notification_period: Name 'message' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in patch_message_middleware: Name 'messages' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_redirects_to_threads: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_get_template_name: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in setUp: Name 'request' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_messages_returned_in_context: Name 'response' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_hide_controls_returned_in_context: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_form_valid: No attribute 'status_code' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 12, in test_dispatch_unauthorized_user: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in create_default_category: Name 'Category' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in location_search: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_sender_thread_is_read: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_correct_count_js_context: No attribute 'user' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_post_no_changes: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_post_returns_correct_count: Name 'url1' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_post_mark_all_read: Name 'thread1' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_with_since: Name 'new_thread' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_with_multiple_id: Name 'response' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_get_previous_release: Can't assign attribute 'return_value' on List[str] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in run: No attribute 'iterations' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 10, in __init__: Function Thread.__init__ was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 22, in run: No attribute 'handle_command' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in loglevel_from_str: Name 'numeric_level' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in log_critical: Function str.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in __init__: Name 'failure_msg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in draw: Name 'b' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in getGameState: Name 'dist' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in __init__: Can't assign attribute 'image' on Tuple[int, int, int] [not-writable]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in __init__: unsupported operand type(s) for *: float and Dict[str, Any] [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in getGameState: No attribute 'pos' on None [attribute-error]
File ""dummpy_input_file"", line 12, in getGameState: No attribute 'height' on None [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in getGameState: Name 'state' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _rngCreepPos: Name 'x' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in set_parent_rules: Name 'rule' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in sub: No attribute 'expand' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in get_substitute: No attribute 'n' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in create_filters: Function yaml.dump_all was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in clean: Name 'line' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 16, in publish_to_ckan: Name 'dataset_entity' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in group_entity_get: Name 'url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __iter__: No attribute 'Mol' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in logout: No attribute 'user' on bool [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_load_config_with_local_config: Can't assign attribute 'return_value' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in __init__: Can't assign attribute 'login_url' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in get_db: Name 'db_engine' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in on_patch: Name 'new_version' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_container_config: Name 'd' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in start_container: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_parser: Name 'parser' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in setup_class: Can't assign attribute 'c' on str [not-writable]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_02_get_api_version: Name 'status' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_11_package_relationship_post: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 24, in publish_file_geodatabase: No attribute 'close' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in jasmine_legacy_done: Name 'failures' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in setUp: Name 'wait' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_delete_button_is_disabled_on_empty_selection: Name 'button' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_delete_button_is_enabled_on_selection: Name 'button' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_select_all_checkbox_sets_on_row_checkboxes: Name 'checkbox' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_on_post_inserts_correct_data: No attribute 'assertEqual' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_on_get_return_correct_data: Name 'result' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 15, in test_on_patch_ok_with_some_fields: No attribute 'mock_req' on Dict[str, str] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 13, in test_on_post_ok: No attribute 'mock_req' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_on_post_start_action_ok: No attribute 'assertEqual' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_on_post_start_action_raises_BadDataFormat_when_job_not_in_session: No attribute 'mock_db' on int [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_on_post_start_succeeds_in_holdoff_if_tag_needs_not_increment: No attribute 'mock_req' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'title' on tuple [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in fields: Name 'locator' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in values: No attribute 'get_attribute' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _get_form_fields: Name 'factory' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: No attribute 'parser' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in handle_create: Name 'client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in handle_create: No attribute 'properties' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _show_resource: Name 'client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_host_list_nova_compute: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 10, in test_panel_not_available: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_update_aggregate_fails_missing_fields: No attribute '_test_generic_update_aggregate' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 36, in test_manage_hosts_update_add_remove_not_empty_aggregate: No attribute 'assertNoFormErrors' on Dict[str, list] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 24, in test_manage_hosts_update_add_not_empty_aggregate_should_fail: No attribute 'hosts' on Dict[str, list] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_manage_hosts_update_add_not_empty_aggregate: Name 'form_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in __init__: Can't assign attribute '_editor' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_search_changes: No attribute 'repo' on None [attribute-error]
File ""dummpy_input_file"", line 7, in test_search_changes: No attribute 'repo' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in mapping_list: Name 'manager' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in handle: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_link_url: Name 'base_url' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_link_url: Name 'base_url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_context_data: Name 'policy' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_context_data: Name 'nat_pool' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in clean: Name 'updated_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in __init__: No attribute 'id' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in handle_create: No attribute 'properties' on Dict[nothing, nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in handle_delete: Name 'svc_profile_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in __init__: No attribute 'id' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in handle: Name 'prid' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in handle: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in handle: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_link_url: Name 'base_url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in patch: Name 'address' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get: Name 'result' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 34, in export_shapefile: No attribute 'close' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 41, in export_csv: No attribute '__iter__' on int [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_file_size: Function genericpath.getsize was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in open_url: No attribute 'last_status' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in package_register_get: Name 'url' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in package_register_post: Name 'headers' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in populate_network_service_policy_id_choices: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_namespace_get_list_kwargs_and_filters: No attribute 'json' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in handle: Name 'msg' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in handle: Name 'volume_type' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in get_agents_data: Name 'network_id' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in _get_data: Name 'network' is not defined [name-error]
File ""dummpy_input_file"", line 12, in _get_data: Name 'network_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 25, in _test_image_create: No attribute 'assertEqual' on dict [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in get_context_data: Name 'context' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in get_context_data: Name 'table' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in delete: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_link_url: Name 'base_url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_text: Name 'text' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_images_pagination: unsupported operand type(s) for item retrieval: Dict[str, Union[bool, int, list]] and int [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 26, in handle: Name 'body' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_image_edit: Name 'request' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_image_get_list_detailed: No attribute 'assertStatusCode' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in rule_list_for_tenant: Name 'rules' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in domain_lookup: Name 'domain' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in get_project_users_roles: Name 'roles' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_update_volume_type: No attribute 'id' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in handle: Name 'project' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _update_project: Name 'e' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _interface_exists: Name 'itfcs_raw' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in _enable_itfcs: No attribute '_check_response' on List[str] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in _get_capabilities: Name 'c' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in _get_running_config: Name 'ioscfg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in _do_remove_vrf: Name 'conf_str' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in _do_add_floating_ip: Name 'conf_str' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _do_remove_floating_ip: Name 'conf_str' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 5, in _get_floating_ip_cfg: Name 'parse' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_simple: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_nested: Name 'dict' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_meaning: No attribute 'meaning' on List[nothing] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 19, in get_head: unsupported operand type(s) for item retrieval: 'possibilities: Iterator[nothing]' and '-1: int' [unsupported-operands]
File ""dummpy_input_file"", line 19, in get_head: Can't assign attribute 'head_phrase' on Iterator [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in get_context_data: No attribute 'submit_url' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in get_data: Name 'routers' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _add_rule: Name 'page' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in fake_init_exception: No attribute 'message' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in add_rule_to_group: Name 'rule_egress_ipv4' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_check_identity_rule_not_found_true: Name 'value' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_image_create_post_copy_from: No attribute '_test_image_create' on Dict[str, Union[bool, str]] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in __init__: unsupported operand type(s) for item retrieval: tuple and str [unsupported-operands]"
False,
True,"File ""dummpy_input_file"", line 26, in clean: Name 'updated_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in __init__: unsupported operand type(s) for item retrieval: tuple and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 28, in handle: unsupported operand type(s) for item retrieval: 'dev_mapping_2: None' and ''delete_on_terminate': str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _filtered_candidates: No attribute 'user_router' on dict [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in _add_internal_nw_nat_rules: No attribute '_nat_rules_for_internet_access' on netaddr.ip.IPAddress [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _get_connection: No attribute '_host_ssh_port' on Exception [attribute-error]
File ""dummpy_input_file"", line 14, in _get_connection: No attribute 'message' on Exception [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in __init__: Can't assign attribute '_timeout' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in find_children: Name 'parent' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in is_openstack_running: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in vdp_uplink_proc_top: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in save_uplink: Name 'args' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_change_password: No attribute 'id' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in test_change_password_validation_for_admin_password: No attribute 'name' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_detail_view: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_detail_view_with_exception: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in run_migrations_online: Name 'connection' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in getLogger: Name 'logger' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in struct: No attribute 'Struct' on Callable[[Any, Any], Any] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in _validate_rt_list: Name 'item' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in is_valid: No attribute 'parent' on List[str] [attribute-error]
File ""dummpy_input_file"", line 9, in is_valid: Built-in function len was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in options: No attribute 'finish' on tuple [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in uuidify: Name 'int_val' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 13, in _ensure_format: Name 'temp_list' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in is_switch_active: Name 'switch_state' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in _get_baremetal_switch_info: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_update_snapshot: No attribute 'id' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 34, in test_edit_stack_template: Name 'stack' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 12, in _test_launch_stack_invalid_name: No attribute 'client' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 23, in test_preview_stack: Name 'form_data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_resource_view: No attribute 'resource_name' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_clean_file_upload_form_multiple: Name 't' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_clean_file_upload_form_invalid_json: No attribute 'SimpleFile' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_update_cgroup_add_vol: No attribute 'mox' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in create_thread: Name 'ipt_thrd' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _is_ip_in_rule: Name 'ip_loc' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in update_vm_result: Name 'msg' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in process_err_queue: Name 'prio' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'shared_net' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in setUp: No attribute '_fake_get_neutron_res' on mock.mock._patch[Union[mock.mock.AsyncMock, mock.mock.MagicMock]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in _fake_get_neutron_res: Name 'res_list' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in get_context_data: No attribute 'render_row_actions' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in test_delete_network_no_subnet: No attribute 'client' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_image_upload_disabled: No attribute 'images' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_launch_stack_with_hidden_parameters: No attribute 'client' on Dict[str, Any] [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_launch_stack_parameter_types: Name 'stack' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_project_delete_event: No attribute 'dfa_server' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 18, in test_network_create_event: Name 'expected_calls' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 37, in test_subnet_create_event: No attribute 'assertTrue' on Dict[str, Dict[str, Any]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in test_dcnm_network_delete_event: No attribute 'dfa_server' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_correct_dhcp_ports: No attribute 'dfa_server' on Dict[str, list] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in test_create_port_chain: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_port_pair_group_update: No attribute 'fmt' on Dict[str, Dict[str, Union[list, str]]] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in delete_routertype: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _make_routertype_dict: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 27, in _inner: Name 'excep' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in get_all_profiles: No attribute 'invoke_api' on List[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in filter_datastores_by_hubs: Name 'hub_ids' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_unset_before: No attribute '__exit__' on logging.Logger [attribute-error]
File ""dummpy_input_file"", line 11, in test_unset_before: No attribute '__enter__' on logging.Logger [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 11, in test_set_before: No attribute 'assertEqual' on logging.Logger [attribute-error]"
False,
False,
True,"File ""dummpy_input_file"", line 9, in expected_wsdl: Name 'path' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _test_lldp_init: Name 'period_obj' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_vdp_port_up_new_nwk: No attribute 'lldpad' on int [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_vdp_port_up_new_nwk_after_restart: Name 'vlan_ret' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_vdp_port_up_new_nwk_invalid_vlan: Name 'vlan_ret' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_vdp_port_down: No attribute 'lldpad' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 20, in _port_action_vlan: Function str.join was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in delete_port_precommit: Name 'vni' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_envvar_prefer_ipv6_override: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_have_envvars: Name 'c' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_vendors_valid_json: No attribute 'assertTrue' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_port_pair_update: Name 'return_value' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in setUp: No attribute 'user_defined_config' on BinaryIO [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 12, in args: No attribute 'parse_args' on argparse._MutuallyExclusiveGroup [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in __init__: Can't assign attribute 'shared_net' on Dict[str, Any] [not-writable]"
False,
True,"File ""dummpy_input_file"", line 4, in __init__: Can't assign attribute 'inject_params' on Dict[str, Any] [not-writable]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in test_container_list_with_bay: Name 'containers' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_container_list_with_marker_limit: No attribute '_test_container_list_with_filters' on List[Tuple[str, str, Dict[nothing, nothing], None]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_container_show: No attribute 'assertEqual' on List[Tuple[str, str, Dict[nothing, nothing], None]] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_container_create: Name 'container' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_container_start: No attribute 'api' on List[Tuple[str, str, Dict[str, str], None]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in _test_arg_success: Name 'stderr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in wrapper: No attribute 'release' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in create_tempfiles: Name 'basename' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_request_ok_with_body: No attribute 'url' on Dict[str, Dict[str, str]] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in translate: Name 'translated_message' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 14, in _translate_msgid: Name 'lang' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in get_rule_id_from_name: No attribute 'get' on None [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in take_action: No attribute 'rule_name' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in take_action: No attribute 'delta' on Dict[str, Any] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 4, in get_parser: Name 'parser' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in do_baymodel_create: No attribute 'flavor_id' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in _load_ip_q: Name '_all_ips' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_no_fallthrough: Name 'c' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in _get_specific_config: No attribute 'split' on Dict[nothing, nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_url_generation_without_trailing_slash_in_base: Name 'url' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_get_connection_params_with_ssl_params: No attribute 'assertEqual' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in take_action: Name '_speaker_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in get_parser: Name 'parser' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in call_server: Name 'data' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 14, in test_stingy_scheduling: No attribute 'assertNotEqual' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in test_extend_hosting_port_info_adds_segmentation_id_internal: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in create_router: Name 'router_dict' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in do_pod_update: Name 'pod' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_list_cfg_agents_handling_hosting_device: Name 'cfg_agent_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in faked_choice: Name 'res' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_get_cfg_agents_for_hosting_devices: Name 'assignable_agents' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_get_cfg_agents_for_hosting_devices_no_schedule: No attribute '_test_get_cfg_agents_for_hosting_devices' on Set[nothing] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 36, in setUp: No attribute '_mock_svc_vm_create_delete' on Dict[nothing, nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_hosting_device_assign_to_cfg_agent: Name 'hosting_device' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_hosting_device_assign_to_non_existing_cfg_agent: unsupported operand type(s) for item retrieval: str and str [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_update_bgp_speaker_exception: No attribute '_test_update_resource' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 7, in test_list_bgp_speaker: No attribute '_test_list_resources' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_list_routes_advertised_by_a_bgp_speaker: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_list_ports_limit: No attribute '_test_list_resources' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_list_router_ports: No attribute '_test_list_router_port' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_update_port_device_id_device_owner: No attribute '_test_update_resource' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in side_effect: No attribute 'value' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_list_bgp_speakers_on_dragent: Name 'path' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_create_ipsecpolicy_with_invalid_lifetime_keys: No attribute '_test_lifetime_values' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 3, in test_create_ipsecpolicy_with_invalid_lifetime_units: No attribute '_test_lifetime_values' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_update_ipsecpolicy: No attribute '_test_update_resource' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_show_vpnservice_id: No attribute '_test_show_resource' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in func: Name 'evt' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 36, in _inner: Name 'e' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 27, in _inner: Name 'e' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in __str__: Name 'string' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 15, in _decode_ddr3: Name 'fineoffset' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_profile: Name 'profile' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 21, in create_keypair: No attribute 'private_key' on TextIO [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in take_action: Name 'data' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in list_profiles: Name 'profile' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 18, in _sendoutput: Function struct.unpack was called with the wrong arguments [wrong-arg-types]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in get_imm_property: unsupported operand type(s) for item retrieval: 'cmdlen: int' and ''data': str' [unsupported-operands]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_update_port_with_no_dns_name: No attribute '_test_update_resource' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 11, in test_update_port_allowed_address_pair_ipaddr: No attribute '_test_update_resource' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_show_port: Name 'args' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 9, in test_delete_port: No attribute '_test_delete_resource' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 40, in setUp: No attribute '_devmgr' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_show_hosting_device_template: No attribute 'get_response' on str [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_list_hosting_device_templates: Name 'hdt2' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in add_arguments_allowedaddresspairs: Name 'group_aap' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_parser: Name 'parser' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in get_parser: Name 'parser' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 8, in take_action: Name 'policy_id' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_remove_router_from_agent: No attribute '_test_remove_from_agent' on Tuple[Any, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 13, in test_create_port_with_binding_profile: No attribute '_test_create_resource' on List[str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in delete_account_metadata: Name 'account' is not defined [name-error]"
False,
False,
True,"File ""dummpy_input_file"", line 4, in create_container: No attribute '_create' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in delete_container_metadata: Name 'res' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_show_flavor_profile: No attribute '_test_show_resource' on list [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in tearDownClass: Name 'snapshot' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_from_config_given_options: No attribute 'assertEqual' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_init: No attribute 'service_keys' on profile.Profile [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in test_set: No attribute 'assertEqual' on profile.Profile [attribute-error]
File ""dummpy_input_file"", line 15, in test_set: No attribute 'get_filter' on profile.Profile [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 5, in test_set_version_bad_service: No attribute 'assertRaises' on profile.Profile [attribute-error]
File ""dummpy_input_file"", line 5, in test_set_version_bad_service: No attribute 'set_version' on profile.Profile [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 8, in test_set_all: Name 'service' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_list: Name 'ids' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 12, in test_basic: No attribute 'assertTrue' on profile.Profile [attribute-error]
File ""dummpy_input_file"", line 12, in test_basic: No attribute 'allow_list' on profile.Profile [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 18, in test_instantiate: No attribute 'created_at' on datetime.datetime [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_token_and_endpoint_before_auth: Name 'token' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 8, in test_token_and_endpoint: Name 'endpoint' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 10, in test_list_all: Name 'api_client' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 4, in add_common_opts: Name 'opt' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in sufficient_options: Name 'missing' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 22, in get_rack_context: No attribute 'ipc_endpoint' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_list: Name 'processes' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in test_create_invalid_parameters: No attribute 'gid' on str [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 6, in test_update: Name 'body' is not defined [name-error]"
False,
True,"File ""dummpy_input_file"", line 4, in test_delete: No attribute 'cs' on str [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 7, in test_basic: No attribute 'assertTrue' on queue.Queue[nothing] [attribute-error]
File ""dummpy_input_file"", line 7, in test_basic: No attribute 'allow_create' on queue.Queue[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 6, in test_make_it: No attribute 'assertEqual' on queue.Queue[nothing] [attribute-error]
File ""dummpy_input_file"", line 6, in test_make_it: No attribute 'name' on queue.Queue[nothing] [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 17, in test_create: No attribute 'service' on Dict[str, str] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_find: Name 'sot' is not defined [name-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_list: No attribute 'RULE_ID' on list [attribute-error]"
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_find: Name 'sot' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in test_system_metadata: Name 'account' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 3, in tearDownClass: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 16, in setUpClass: Name 'sot' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 5, in add_type_hours: Name 'cts' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 17, in test_make_it: No attribute 'updated_at' on datetime.datetime [attribute-error]"
False,
True,"File ""dummpy_input_file"", line 15, in test_list: Name 'result' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 9, in test_get: Name 'mgr' is not defined [name-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in get: No attribute '_get' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
True,"File ""dummpy_input_file"", line 2, in json: No attribute 'loads' on Callable[[Any], Any] [attribute-error]
File ""dummpy_input_file"", line 2, in json: No attribute 'text' on Dict[str, Any] [attribute-error]"
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
False,
